InternalPythonModules/android/viber.py

```diff
@@ -117,13 +117,28 @@ def parse_contacts(self, contacts_db, helper):
         try:
             contacts_parser = ViberContactsParser(contacts_db)
             while contacts_parser.next():
-                helper.addContact( 
-                    contacts_parser.get_contact_name(), 
-                    contacts_parser.get_phone(),
-                    contacts_parser.get_home_phone(),
-                    contacts_parser.get_mobile_phone(),
-                    contacts_parser.get_email()
-                )
+                if contacts_parser.get_phone() is not None:
```

Comment:
If this value is a String, I think we should check for blankness too. Blankness can be described as the following conditions:

is_blank(None) //True
is_blank("") //True
is_blank("   ") //True

Reply:
using isspace() to check for "", " ", and special characters as well as None